appnexus/lucid
# Bugfix/TARG-917: Prevent user from pressing enter on SearchableMultiSelect component

## PR Checklist

Current Storybook can be viewed [here](https://docspot.adnxs.net/projects/lucid/latest/?path=/docs/controls-searchablemultiselect-sb5--basic)

Proposed Storybook can be viewed [here](https://docspot.adnxs.net/projects/lucid/bugfix_TARG-917/?path=/docs/controls-searchablemultiselect-sb5--basic)

- Manually tested across supported browsers




When pressing enter after typing into the `SearchField` a selection is made that is not relevant to the search text. Please see the gif below for an example. 'Maine' is typed in and Louisiana is the option that is selected.

Fix: Prevent the user from hitting enter at all in the SearchableMultiSelect component. Have the user click specific choices. 


**Before**
![ezgif com-gif-maker (2)](https://user-images.githubusercontent.com/21188892/139470946-670c3cf3-3203-4a5e-9dc4-1776a68a5749.gif)


**After**
![ezgif com-gif-maker](https://user-images.githubusercontent.com/21188892/139470762-279cec39-1154-4c3a-ad59-31e4deca4e45.gif)
